sycl/test-e2e/lit.cfg.py

```diff
@@ -284,6 +284,11 @@ def open_check_file(file_name):
 else:
     config.substitutions.append(("%level_zero_options", ""))
 
+dx12_options = (" -l d3d12 -l dxgi -l dxguid")
```

Comment:
why need a space at the beginning of the string?

Reply:
I follow other substitution formats like "cuda_options" and "level_zero_options"